pkg/binary/envoy/controlplane/istio_test.go

```diff
@@ -76,6 +77,10 @@ func setupMockPilot() (*bootstrap.Server, util.TearDownFunc) {
 		args.Config.FileDir = "testdata"
 		args.Plugins = bootstrap.DefaultPlugins
 		args.Mesh.MixerAddress = ""
+		// In a normal macOS setup, you cannot write to /dev/stdout, which is the default path here.
+		// While not Docker-specific, there are related notes here https://github.com/moby/moby/issues/31243
+		// Since this test doesn't read access logs anyway, the easier workaround is to disable access logging.
+		args.MeshConfig.AccessLogFile = ""
```

Comment:
Key fix for darwin is here, and when there's a better solution (than running as root or changing host perms), hopefully this comment explains enough.